dashboard/assets/common.jsx

```diff
@@ -69,3 +69,11 @@ export const styles = {
 		color: 'rgba(255, 255, 255, 0.54)',
 	},
 };
+
+// htmlEscape replaces the HTML specific special characters in a string.
+export const htmlEscape = (str: string) => str
+	.replace(/&/g, '&amp;')
+	.replace(/"/g, '&quot;')
+	.replace(/'/g, '&#39;')
+	.replace(/</g, '&lt;')
+	.replace(/>/g, '&gt;');
```

Comment:
This feels like reinventing the wheel, possibly in an unsecure way. I'm certain there's a javascript function which already does safe escaping. Lets please call that instead.